src/common/types.ts

```diff
@@ -121,3 +121,8 @@ export interface ILoggerWrapper {
   info(message: any, ...args: any[]): any;
   debug(message: any, ...args: any[]): any;
 }
+
+export interface IUserRecord {
+  count: number;
+  date: string;
+}
```

Comment:
Mongo can take `Date` objects.

Reply:
Also add `guild: Snowflake` field. Makes it much easier to distinguish between servers (prod and dev servers)